orderer/common/channelparticipation/restapi.go

```diff
@@ -75,7 +80,7 @@ func NewHTTPHandler(config localconfig.ChannelParticipation, registrar ChannelMa
 		router:    mux.NewRouter(),
 	}
 
-	// swagger:operation GET /v1/participation/channels/{channelID} channels listChannel
+	// swagger:operation GET /participation/v1/channels/{channelID} channels listChannel
```

Comment:
why did you change it?

Reply:
According to the code, everywhere orderer and osnadmin we use ["/participation/v1/"](https://github.com/hyperledger/fabric/blob/main/orderer/common/channelparticipation/restapi.go#L29). And only in the swagger file the path is reversed "/v1/participation/". This option is not working. This means that no one has used the swagger file yet.
Of the two options, I chose to leave the one that is exactly being used, respectively, and fixed the other one.

Reply:
The description of `// swagger:operation` only helps the program generate the file swagger.json doesn't affect anything else. 
The actual path pattern is defined below, and usually uses the constant `URLBaseV1`